WordPress/Classes/ViewRelated/NUX/SignupGoogleViewController.swift

```diff
@@ -47,6 +60,8 @@ extension SignupGoogleViewController: GIDSignInDelegate {
                 self.navigationController?.popViewController(animated: true)
                 return
         }
+        googleUser = user
+
         NSLog(token)
```

Comment:
Oh damn, this `NSLog` should be gone. My bad. Remove it if you wanna, otherwise, I gotta remember to…

Reply:
I got it!